### DISCUSSION TITLE
Pull request

### DISCUSSION THREAD

**Discussion by IzmaDoesItbeta** [community/community#116867] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

New Trust Certificates currently have bad ones with Malware.

**Comment by Akash1134** [comment]
Hi @IzmaDoesItbeta , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @IzmaDoesItbeta! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
